Explorer/Assets/DCL/Landscape/Assets/Wind.prefab

```diff
@@ -41,7 +41,7 @@ WindZone:
   m_GameObject: {fileID: 4984400270765111023}
   m_Enabled: 1
   m_Mode: 1
-  m_Radius: 120
+  m_Radius: 40
```

Comment:
I don't understand why this would do anything. It's not like the game swaps to a cheaper shader when outside the wind zone, is it?

Reply:
On a very quick test, it did seem that it was able to keep/improve performance with a lower radius. My guess was that if the trees are outside the radius, then the caluclation was ignored alltogether.

I would need to analyze it a little bit further to understand it; but given that this is going away really soon, Im not sure its worth it. I ll see if I can allocate some time to it.